catalog/video/catalog-mpegts-transformer/src/main/java/com/connexta/transformer/video/Constants.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformer.video;
+
+public class Constants {
```

Comment:
Why is this class public?  It looks like these constants are an implementation detail that should be hidden from client code.  Can these be broken up and defined in the classes that use them instead?